pkgs/google_generative_ai/pubspec.yaml

```
@@ -11,7 +11,7 @@ environment:
 
 dependencies:
   http: ^1.1.0
-  pool: ^1.5.0
+  meta: ^1.11.0
```

This should address the CI failure:

```suggestion
  meta: ^1.10.0
```